src/Tribe/Views/V2/Customizer.php

```diff
@@ -363,9 +363,20 @@ public function filter_global_elements_css_template( $css_template, $section, $c
 					background-color: <%= global_elements.accent_color %>;
 				}
 
-				.tribe-events-widget .tribe-events-widget-events-list__view-more-link {
+				.tribe-common.tribe-events-widget .tribe-events-widget-events-list__view-more-link {
```

Comment:
curious why we need the extra selector. was it because you had visited the link and it was applying the `:visited` style? or are the customizer styles being loaded before the stylesheets?

Reply:
Because the theme styles are overriding it otherwise: https://d.pr/i/yyzwmb (2021 theme)
This happens in 3 of the last 4 official themes (2017, 2019, 2021). I've refactored to be theme overrides.